packages/rollup-plugin/src/index.ts

```diff
@@ -210,26 +229,27 @@ export function stylableRollupPlugin({
             };
 
             for (const moduleId of this.getModuleIds()) {
-                if (moduleId.endsWith(ST_CSS)) {
-                    modules.push({ depth: calcDepth(moduleId, context, [], cache), moduleId });
+                const { isStFile, isLoadableCssFile, path } = getLoadableModuleData(moduleId);
+                if (isStFile || isLoadableCssFile) {
+                    modules.push({ depth: calcDepth(moduleId, context, [], cache), path });
                 }
             }
 
             sortModulesByDepth(
                 modules,
                 (m) => m.depth,
-                (m) => m.moduleId,
+                (m) => m.path,
                 PRINT_ORDER
             );
 
             outputCSS = '';
 
-            for (const { moduleId } of modules) {
-                const stored = extracted.get(moduleId);
+            for (const { path } of modules) {
```

Comment:
I'm not sure all modules have path...

Reply:
they do because we create the module list in this scope and the path is created from the module id minus any query params